config/webpack/loaders/vue.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Definition for rule 'vue/max-len' was not found  vue/max-len<br>'module' is not defined                          no-undef